### ISSUE TITLE
Matplotlib vs. Holoviews

### ISSUE THREAD

**Issue by grst** [scverse/plotting_api#12]
[HoloViews](https://holoviews.org/index.html) abstracts multiple backends, including matplotlib. 
How much of the logic we prototyped here (dealing with gridspecs, distinguishing between notebook and console case, ...) would we still require when switching to HoloViews? 

@flying-sheep shall we try to reproduce what we built here with holoviz tools?

**Comment by flying-sheep** [comment]
Yeah! I think there are a few plot types missing from HoloViews, so we should try rebuilding plots too and report what we’d like to see.
